src/CloudFlare/Zone/Settings.php

```diff
@@ -471,6 +471,8 @@ public function change_ipv6($zone_identifier, $value = 'off') {
 	 * @return array|mixed
 	 */
 	public function change_minify($zone_identifier, $value) {
+		
+		$data = ['value' => $value];
```

Comment:
I'm confused.  $data isn't used and $value already is below.  What are you trying to fix with this PR?